README.md

```diff
@@ -4,10 +4,11 @@ This is a simple example to show how to write a uVisor-secured threaded applicat
 
 Supported devices:
 
-| Target         | Toolchain | Public box LED | Secure box LED | User button   | Baud rate |
-|----------------|-----------|----------------|----------------|---------------|-----------|
-| `K64F`         | `GCC_ARM` | `LED_BLUE`     | `LED_RED`      | `SW2`         | 9600      |
-| `DISCO_F429ZI` | `GCC_ARM` | `LED1`         | `LED2`         | `USER_BUTTON` | 9600      |
+| Target            | Toolchain | Public box LED | Secure box LED | User button   | Baud rate |
+|-------------------|-----------|----------------|----------------|---------------|-----------|
+| `K64F`            | `GCC_ARM` | `LED_BLUE`     | `LED_RED`      | `SW2`         | 9600      |
+| `DISCO_F429ZI`    | `GCC_ARM` | `LED1`         | `LED2`         | `USER_BUTTON` | 9600      |
+| `EFM32GG_STK3700` | `GCC_ARM` | `LED1`         | `LED2`         | `SW0`         | 9600      |
```

Comment:
In the commit message that adds EFM32GG, please mention that you fix https://github.com/ARMmbed/mbed-os-example-uvisor/issues/18 to auto-close it.